tests/union_duplicates.pyi

```diff
@@ -43,3 +43,14 @@ f: Literal["foo"] | Literal["bar"] | int | float | builtins.bool  # Y030 Multipl
 MyTypeAlias: TypeAlias = int | float | bool
 MySecondTypeAlias: TypeAlias = Union[builtins.int, str, complex, bool]
 MyThirdTypeAlias: TypeAlias = Mapping[str, int | builtins.float | builtins.bool]
+
+one: str | Literal["foo"]  # Y051 "Literal['foo']" is redundant in a union with "str"
+Two: TypeAlias = Union[Literal[b"bar", b"baz"], bytes]  # Y051 "Literal[b'bar']" is redundant in a union with "bytes"
+def three(arg: Literal[5] | builtins.int | Literal[9]) -> None: ...  # Y030 Multiple Literal members in a union. Combine them into one, e.g. "Literal[5, 9]".  # Y051 "Literal[5]" is redundant in a union with "int"
```

Comment:
For completeness sake maybe add some variants with multiple things inside Literal, like `Literal[3, "x"] | int` or `Literal[b"x", 3] | int`.